github_pr.py

```diff
@@ -369,8 +395,16 @@ def main():
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+
+Check conditional status checks
+
+    github-pr check-condition -r dataxu/dcommand -n 84 --condition-owner-cannot-ship
+        This will check to make sure that the owner can not apply a shipable comment on their own code
+        Also available at this time is a check to make the comment is from an approved shipper
+        This check only looks at comments AFTER the latest commit, to validate that the
+        code has been most recently peer reviewed.
```

Comment:
right, we did want to change that name, "--condition-found-approved-merger" sounds good, or maybe "--condition-approved-merger-required" to more closely match the restrictive phrasing of "--condition-owner-cannot-ship"?

I added the doc/example for "--condition-approved-mergers" because we have it implemented, even if we're not using it and don't expect it to be used much. We could mention it like this:
```
github-pr check-condition -r dataxu/dcommand -n 84 --condition-approved-mergers "ned_flanders,marge_simpson"
    This will check that a comment containing a :shipit: (the default, or other defined comment if merge_comment is set) comes from a user in the provided list passed on the commandline. A MAINTAINERS file is the preferred convention to use, and this option is primarily for local testing when setting up your CD flow.
```